Blaise-Shyaka/AnnounceIT
# Ft signup page #170772164

# What does this PR do?

This is a page allowing users to create an account on AnnounceIT

# Description of tasks to be completed

- Create a simple, responsive signup page
- It should collect at least the first name, last name, email, and password of the user

# How should this be manually tested?

- Clone this repository
- cd into /templates/signup.html
- Resize the browser to check its responsiveness
- Check the presence of form fields to fill in the first name, last name, emain, and password
